source/whats-new.txt

```diff
@@ -64,8 +67,18 @@ The 5.0 driver release introduces the following features:
   This relates to a :ref:`breaking change about Observables in this version
   <java-breaking-changes-v5.0-observables>`.
 
-- Changes to the ``ConnectionId`` class. To learn more, see the
-  :ref:`Version 5.0 Breaking Changes <java-breaking-changes-v5.0>`.
+- The first parameter, time durations, of the following ``SocketSettings``
+  builder methods is of ``long`` type:
```

Comment:
Q: is the first parameter called "time durations"? If so I would monospace it (and I'm assuming camel-case it). I think being more specific with it will be more helpful.

Also I think we could be more concise here by just saying:

```suggestion
- The time durations parameter of the following ``SocketSettings``
  builder methods is of ``long`` type:
```

Reply:
The parameter names are `connectTimeout` and `readTimeout`, which is the first parameter of each. I've changed to your phrasing and changed it to "timeout", but have not monospaced it. I think it should be clear this way to anyone who's using either method.